.github/workflows/call-manage-issue-header.yml

```diff
@@ -0,0 +1,13 @@
+name: Manage issue header
+
+on:
+  issues:
+    types: [opened, labeled, unlabeled]
```

Comment:
Just to cover some edge cases we can add `reopened` and `transferred` to this list.